examples/Component/Constraint/Lagrangian/FrictionContact_LCP_with_friction.scn

```diff
@@ -0,0 +1,64 @@
+        <RequiredPlugin name="Sofa.Component.Constraint.Lagrangian.Solver"/> <!-- Needed to use components [LCPConstraintSolver] -->
+        <RequiredPlugin name="Sofa.Component.IO.Mesh"/> <!-- Needed to use components [MeshOBJLoader] -->
+        <RequiredPlugin name="Sofa.Component.LinearSolver.Iterative"/> <!-- Needed to use components [CGLinearSolver] -->
+        <RequiredPlugin name="Sofa.Component.Mapping.NonLinear"/> <!-- Needed to use components [RigidMapping] -->
+        <RequiredPlugin name="Sofa.Component.Mass"/> <!-- Needed to use components [UniformMass] -->
+        <RequiredPlugin name="Sofa.Component.ODESolver.Backward"/> <!-- Needed to use components [EulerImplicitSolver] -->
+        <RequiredPlugin name="Sofa.Component.StateContainer"/> <!-- Needed to use components [MechanicalObject] -->
+        <RequiredPlugin name="Sofa.Component.Topology.Container.Constant"/> <!-- Needed to use components [MeshTopology] -->
+        <RequiredPlugin name="Sofa.Component.Visual"/> <!-- Needed to use components [VisualStyle] -->
+        <RequiredPlugin name="Sofa.GL.Component.Rendering3D"/> <!-- Needed to use components [OglModel] -->
+    </Node>
+    <VisualStyle displayFlags="showVisual" />
+
+    <LCPConstraintSolver tolerance="1e-6" maxIt="1000" initial_guess="true" build_lcp="false" mu="0.2"/>
+    <FreeMotionAnimationLoop />
+    <CollisionPipeline depth="15" verbose="0" draw="0" />
+    <BruteForceBroadPhase/>
+    <BVHNarrowPhase/>
+    <LocalMinDistance name="Proximity" alarmDistance="0.3" contactDistance="0.1" useLMDFilters="0" />
+    <DefaultContactManager name="Response" response="FrictionContactConstraint" responseParams="mu=0.2" />
+    <Node>
+        <EulerImplicitSolver name="cg_odesolver" printLog="false"  rayleighStiffness="0.1" rayleighMass="0.1" />
+        <CGLinearSolver iterations="25" name="linear solver" tolerance="1.0e-9" threshold="1.0e-9" />
+        <Node name="CUBE_1_1">
+            <MechanicalObject template="Rigid3" scale="0.3" dx="-2.8" dy="-1.5" dz="0.0" rx="0" />
+            <UniformMass totalMass="100.0" />
+            <UncoupledConstraintCorrection useOdeSolverIntegrationFactors="0"/>
+            <Node name="Visu">
+                <MeshOBJLoader name="meshLoader_4" filename="mesh/smCube27.obj" scale="0.3" handleSeams="1" />
+                <OglModel name="Visual" src="@meshLoader_4" color="0.0 0.5 0.8 1.0" />
+                <RigidMapping input="@.." output="@Visual" />
+            </Node>
+            <Node name="Surf2">
+                <MeshOBJLoader name="loader" filename="mesh/smCube27.obj" triangulate="true" />
+                <MeshTopology src="@loader"/>
+                <MechanicalObject src="@loader" scale="0.3" />
+                <TriangleCollisionModel />
+                <LineCollisionModel />
+                <PointCollisionModel />
```

Comment:
If we consider that it is allowed to use plugins components in an example scene, you can use `OBBCollisionModel`
```suggestion
                <TriangleCollisionModel />
                <LineCollisionModel />
                <PointCollisionModel />
```

Reply:
Actually, I just took inspiration from the FrictionContact_VelocityConstraints.scn scene 😅
And I dont think it is a good idea to use plugin in the "main trunk" of SOFA scenes